resources/phong_frag.glsl

```diff
@@ -34,8 +35,41 @@ uniform float MatShiny;
 uniform mat4 M;
 uniform mat4 V;
 
+uniform vec2 shadowMapSize;
+
+uniform float PCFkernelRadius = 4.0;
+
+//Poisson Disc for PCF filtering
+uniform vec2 poission[25] = { vec2(-0.978698, -0.0884121),
```

Comment:
Poission Disc Values to evaluate only some texels in the PCF filter kernel, not all. Precomputed for max render speed.